app/gui/view/documentation/assets/stylesheet.css

```diff
@@ -0,0 +1,565 @@
+/*
+! tailwindcss v3.2.4 | MIT License | https://tailwindcss.com
+*/
+
+/*
+1. Prevent padding and border from affecting element width. (https://github.com/mozdevs/cssremedy/issues/4)
+2. Allow adding a border to an element by just adding a border-width. (https://github.com/tailwindcss/tailwindcss/pull/116)
```

Comment:
Is this file meant to be gitignored? Because it is mentioned in gitignore but its committed

Reply:
It is not gitignored. It is only mentioned in `.prettierignore`, as it is auto-generated. Keeping this file in VCS is a deliberate decision because, without it, the IDE (or, rather, the documentation panel) won't work correctly. It doesn't make much sense to regenerate it every build.